FinSearchUnified/FinSearchUnified.php

```diff
@@ -54,6 +56,29 @@ public function install(InstallContext $context)
     {
         $context->scheduleClearCache([InstallContext::CACHE_TAG_THEME]);
         parent::install($context);
+
+        $shopwareVersion = Shopware()->Config()->get('version');
+
+        if (version_compare($shopwareVersion, '5.2.17', '>=')) {
+            $file = $this->getPath() . '/Resources/config/config.xml';
+        } else {
+            // For Shopware <= 5.2.17 we load a different config file due to missing `button` element in configuration
+            $file = $this->getPath() . '/Resources/config/config_compat.xml';
+        }
+
+        if (class_exists(XmlConfigReader::class)) {
+            $xmlConfigReader = new XmlConfigReader();
+        } else {
+            $xmlConfigReader = new \Shopware\Components\Plugin\XmlConfigDefinitionReader();
```

Comment:
Import this class.

Reply:
Importing this class was trying to find the class from the current namespace, which was failing the test. So I then used FQN instead.